scrapely/extraction/regionextract.py

```diff
@@ -359,13 +371,17 @@ def _doextract(self, page, region_elements, start_index, end_index, nested_regio
         nested_regions = nested_regions or []
         ignored_regions = ignored_regions or []
         first_region, following_regions = region_elements[0], region_elements[1:]
-        while following_regions and labelled_element(following_regions[0]).start_index \
-                < labelled_element(first_region).end_index:
+        while (following_regions and
+               _int_cmp(labelled_element(following_regions[0]).start_index,
+                        labelled_element(first_region).end_index)):
```

Comment:
I'm too picky today :) Just '_int_cmp' reads as 'equals?', not as 'less than'. What about removing the default value for 'op' argument of _int_cmp function? Or maybe you have another suggestions about how to make the code more readable?